index.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Capstone</title>
+</head>
+<body>
+    <section class="container">
+        <header>
+            <nav>
+                <div class="desktop-nav">
+                    <img class="facebook-icon" src="./images/base/facebook.png" alt="">
+                    <img class="twitter-icon" src="./images/base/twitter.png" alt="">
+                    <ul>
+                        <li><a href="">English</a></li>
+                        <li><a href="">My Page</a></li>
+                        <li><a href="">Logout</a></li>
+                    </ul>
+                </div>
+                <div class="desktop-navbar">
+                    <a class="logo" href="">HUDSON</a> 
+                    <ul class="desktop-nav-items">
+                        <li class="nav-items"><a href="about.html">About</a></li>
+                        <li class="nav-items"><a href="">Services</a></li>
+                        <li class="nav-items"><a href="">Sponsors</a></li>
+                    </ul>
+                </div>
+                <div class="menu-bar">
+                    <div class="burger" id="btn">
+                        <span class="bar" id="btn1"></span>
+                        <span class="bar"></span>
+                        <span class="bar" id="btn2"></span>
+                    </div>
+                </div>
+            </nav>
+        </header>
+        
+        <div class="heading">
+            <p class="title">Making Differences in the life of others</p>
+            <h1>Leading the Way in Medical Excellence</h1>
+            <p class="description">
+                Empowering Lives Through Exceptional Medical Care,
+                Journey to Health and Welliness starts Here.
+            </p>
+            <h2>2015.10.15(THU) ~ 16(FRI)</h2>
+            <p class="second-title">
+                Patience is the road to wisdom.
+            </p>
+        </div>
+    </section>
+
+    <!---Popular Services section--->
+    <section class="main-program">
+        <h1>Popular Services</h1>
+        <div class="cards-container">
+            <div class="card">

```

Comment:
- [ ] [REQUIRED] Good job so far! Your `Main Program` section should have `5` cards but you have `4`, and please give desktop version style to `Learn More` button because it steel displayed like mobile design 🙏🏼 

![image](https://github.com/brighton0102/HTML-CSS-JavaScript-Capstone-Project/assets/63915024/c66d82e2-73f3-4cee-82fc-222194f04673)